services/gitlab/gitlab-release.service.js

```diff
@@ -38,58 +39,44 @@ export default class GitLabRelease extends GitLabBase {
     queryParamSchema,
   }
 
-  static examples = [
-    {
-      title: 'GitLab Release (latest by date)',
-      ...commonProps,
-      queryParams: { sort: 'date', date_order_by: 'created_at' },
-      staticPreview: renderVersionBadge({ version: 'v2.0.0' }),
-    },
-    {
-      title: 'GitLab Release (latest by SemVer)',
-      ...commonProps,
-      queryParams: { sort: 'semver' },
-      staticPreview: renderVersionBadge({ version: 'v4.0.0' }),
-    },
-    {
-      title: 'GitLab Release (latest by SemVer pre-release)',
-      ...commonProps,
-      queryParams: {
-        sort: 'semver',
-        include_prereleases: null,
+  static openApi = {
+    '/gitlab/v/release/{project}': {
+      get: {
+        summary: 'GitLab Release',
+        description: documentation,
+        parameters: [
```

Comment:
In situations where we've got a mix of path and query params, I think an array of calls to `pathParam()` and `queryParam()` is a bit nicer than concatenating the results of a call to `pathParams()` and a call to `queryParams()`